variants/uPesy_edu_esp32/pins_arduino.h

```diff
@@ -0,0 +1,58 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+
+#include <stdint.h>
+
+#define EXTERNAL_NUM_INTERRUPTS 16
+#define NUM_DIGITAL_PINS        40
+#define NUM_ANALOG_INPUTS       16
+
```

Comment:
Those #defines are no longer needed in pins_arduino.h file. They are provided globally per SOC.
```suggestion
```